docs/1_install.md

```diff
@@ -1,9 +1,3 @@
----
-title: Install Crossplane
-toc: true
-weight: 220
-indent: true
----
```

Comment:
Will these nav bar entries be reinstituted at some future point?

Reply:
Yes! I just removed them because they have always been somewhat confusing to me (I am not entirely clear on how Jekyll handles `weight` because I haven't looked into it at all), so I thought it better to just add them back at the end. I like that the file names now indicate the order as well because it is easier to envision when looking at the entire directory.